app/dashboard/utils.py

```diff
@@ -253,6 +258,9 @@ def get_bounty(bounty_enum, network):
 
 # processes a bounty returned by get_bounty
 def web3_process_bounty(bounty_data):
+    if not bounty_data or (settings.DEBUG or settings.ENV != 'prod') and bounty_data.get('network') == 'mainnet':
```

Comment:
might be worth dropping a comment or print statement here s aying what this is for for a future maintainer?

Reply:
also.. `sync_geth` is the main way that people get data into their locals... and this will break that.

Reply:
+1 for enabling rinkeby bounties.

Reply:
This won't stop syncing anything other than `mainnet` bounties on non-prod environments.